app/models/manageiq/providers/vmware/cloud_manager.rb

```diff
@@ -2,6 +2,7 @@ class ManageIQ::Providers::Vmware::CloudManager < ManageIQ::Providers::CloudMana
   require_nested :RefreshParser
   require_nested :RefreshWorker
   require_nested :Refresher
+  require_nested :Template
```

Comment:
I am curious about the `VirtualTemplate` - what is it about and should it be here as well? I saw several providers have them, but Azure doesn't.